hudi-spark-datasource/hudi-spark2/src/main/java/org/apache/hudi/internal/HoodieDataSourceInternalWriter.java

```diff
@@ -89,6 +90,7 @@ public void onDataWriterCommit(WriterCommitMessage message) {
 
   @Override
   public void commit(WriterCommitMessage[] messages) {
+    sparkSession.sparkContext().setJobGroup(this.getClass().getSimpleName(), "Committing to data table", true);
```

Comment:
Same comment: what's this annotating? There's no Spark jobs being executed at this point